modules/cert-manager-acme-challenges-types.adoc

```diff
@@ -12,6 +12,13 @@ HTTP-01:: With the HTTP-01 challenge type, you provide a computed key at an HTTP
 +
 For more information, see link:https://cert-manager.io/docs/configuration/acme/http01/[HTTP01] in the upstream cert-manager documentation.
 
+[NOTE]
```

Comment:
Why the choice to make this a note rather than incorporating the text into the body of the module?

Reply:
It is supposed to be useful and additional information that users should know and expect while going through the description. And also, the requirement was initially to add a note too.